### PR TITLE
Do not remove dash character from a Github username

### PR DESCRIPTION
**Pull request by tommi-lew**

### CODE REVIEW

**Comment by tommi-lew** [comment]
Hi @taylorlapeyre , a small change here to make sure dashes are not gsub-ed away when detecting mentions in a comment.

File: lib/lita/handlers/github_pinger.rb [hunk 1]
```diff
@@ -139,7 +139,7 @@ def act_on_comment(body, response)
           puts "Found @mentions in the body of the comment! Extracting usernames... "
 
           # get each @mentioned engineer in the comment
-          mentions = comment.split("@")[1..-1].map { |snip| snip.split(" ").first.gsub(/[^0-9a-z ]/i, '') }
+          mentions = comment.split("@")[1..-1].map { |snip| snip.split(" ").first.gsub(/[^0-9a-z\- ]/i, '') }
```
Comment (dirk) [inline lib/lita/handlers/github_pinger.rb:142]: @tommi-lew: The `\` escaping of the `-` isn't necessary, see this [Rubular example](http://rubular.com/r/RpHUAddEoR). 😉

**Comment by dirk** [comment]
@tommi-lew: Thanks for the contribution! One [small comment](https://github.com/Everlane/lita-github-pinger/pull/2#discussion_r74697436) for you to address, and then I'd be happy to merge this.

File: lib/lita/handlers/github_pinger.rb (hunk 1 above)
Comment (dirk) [inline lib/lita/handlers/github_pinger.rb:142]: Never mind! I was thinking of `-`'es in a different place in the `Regexp`.

**Comment by dirk** [comment]
@tommi-lew: Thanks for the contribution. Merged. 😄

**Comment by dirk** [comment]
@taylorlapeyre: Want to cut a new version of the gem for this? I also reformatted this code and added a few example comments to make the logic clearer in f84d08c.

**Comment by taylorlapeyre** [comment]
yep!

**Comment by taylorlapeyre** [comment]
Released in `0.6.6`

**Comment by tommi-lew** [comment]
Hi @dirk , I am sorry that I did not respond to your comment on time.  I have been filtering Github notifications away from my email inbox all the while by accident. Anyway, thanks both for merging and releasing :)

**Pull request merged**